docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are implemented based on the Fluid service it's connecting to. This means that the specifics of authentication will differ based on the Fluid service.
+
+The information below is based on Azure Fluid Relay service but also applies to
+[Tinylicious]({{< relref "tinylicious.md" >}}). Other Fluid services may differ.
+
+## Azure Fluid Relay service
+
+{{< include file="_includes/frs-onboarding.html" safeHTML=true >}}
+
+Each FRS tenant you create is assigned a *tenant ID* and its own unique *tenant secret key*.
+
+The secret key is a *shared secret*. Your app/service knows it, and FRS knows it. This means that you can sign data
+using that secret key, and FRS can verify that it is you who signed those requests because it also has that key.
+
+In summary, the secret key is how the Azure Fluid Relay service knows that requests are coming from your app or service. This is critical, because once the Azure Fluid Relay service can trust that it's *your app* making the requests, it can trust the data you send. This is also why it's important that the secret is handled securely. **Anyone with access to the secret can impersonate your application when communicating with Azure Fluid Relay.**
+
+Now you have a mechanism to establish trust. You can sign some data, send it to FRS, and FRS can validate whether the
+data is signed properly, and if so, it can trust it. Fortunately, there's an industry standard way method for encoding
+authentication and user-related data with a signature for verification: JSON Web Tokens (JWTs).
+
+{{% callout note %}}
+
+The specifics of JWTs are beyond the scope of this article. For more details about the JWT standard see
+<https://jwt.io/introduction>.
+
+{{% /callout %}}
+
+JSON Web Tokens are a signed bit of JSON data that can include additional data about the rights conferred by the
+JWT. The Azure Fluid Relay service uses signed JWTs for establishing trust with calling clients.
+
+The next question is: what data should you send?
+
+You need to send your *tenant ID* so that FRS can look up the right secret key to validate your request. You need to
+send the *container ID* (called `documentId` in the JWT) so FRS knows which container the request is about. Finally, you
+need to also set the *scopes (permissions)* that the request is permitted to use -- this allows you to establish your
+own user permissions model if you wish.
```

Comment:
I think the phrasing might be better if it's more like you need to sign these things and provide it to the `frs-client` where it will do the right thing.

Reply:
Or maybe it's actually you need to write a token provider... But either way the developer is not responsible for sending things. Simply including them to be sent.